docker-compose-bin.yml

```diff
@@ -0,0 +1,11 @@
+version: "2.1"
```

Comment:
@josephharrington Docker Compose has a number of `docker-compose.yml` file formats with particular Docker Engine version dependencies. The following explains compatibility requirement and API versioning with upgrade process in details:
- https://docs.docker.com/compose/compose-file/compose-versioning/#compatibility-matrix
- https://docs.docker.com/compose/compose-file/compose-versioning/#versioning

At the time of writing compose file for ClusterRunner in this pull request, Docker version in my CoreOS environment supported compose API version 2.1.